lua/cinema/entities/magicbutton.lua

```diff
@@ -304,7 +304,11 @@ function ENT:Draw()
 
     render.SetAmbientLight(0.05, 0.05, 0.05)
     --render.SuppressEngineLighting( true )
-    self:DrawModel()
+    if util.IsValidModel("models/pyroteknik/secretbutton.mdl") then
+        self:DrawModel()
+    else
```

Comment:
the else/setnodraw is pointless. if the if check fails, you don't draw the model.

Reply:
I added it because it was still drawing a shadow when i was testing it on theatron

Reply:
https://i.imgur.com/AqyHB8w.png lol